mkdocs.yml

```diff
@@ -9,27 +9,144 @@ remote_branch: gh-pages
 
 nav:
   - 'Overview': index.md
-  - 'Custom Projections': custom_projections.md
-  - 'Query Arguments': query_arguments.md
-  - 'Types': types.md
-  - 'Transactions': transactions.md
-  - 'Custom Column Types': custom_column_types.md
-  - 'Migrations': migrations.md
-  - 'RxJava': rxjava.md
-  - 'Coroutines': coroutines.md
-  - 'Multiplatform': multiplatform.md
-  - 'Android Paging': android_paging.md
-  - 'Supported Dialects': supported_dialects.md
-  - 'IntelliJ Plugin': intellij_plugin.md
-  - 'Gradle': gradle.md
-  - '1.x API':
-      - 'coroutines-extensions': 1.x/coroutines-extensions/index.html
-      - 'native-driver': 1.x/native-driver/index.html
-      - 'runtime': 1.x/runtime/index.html
-  - 'Upgrading': upgrading.md
   - 'Changelog': changelog.md
   - 'Contributing': contributing.md
-  - 'Code of Conduct': code_of_conduct.md
+
+  - 'Android':
+    - 'Getting Started': android_sqlite/index.md
+    - 'SQL':
+      - 'Projections': android_sqlite/custom_projections.md
+      - 'Arguments': android_sqlite/query_arguments.md
+      - 'Types': android_sqlite/types.md
+      - 'Transactions': android_sqlite/transactions.md
+    - 'Extensions':
+        - 'RxJava': android_sqlite/rxjava.md
+        - 'Coroutines': android_sqlite/coroutines.md
+        - 'Android Paging': android_sqlite/android_paging.md
+    - 'Migrations': android_sqlite/migrations.md
+    - 'IntelliJ Plugin': android_sqlite/intellij_plugin.md
+    - 'Gradle': android_sqlite/gradle.md
+    - 'Upgrading Pre-1.0': android_sqlite/upgrading.md
+    - '1.x API':
```

Comment:
neither of the links in the entire `1.x API` section are working (in each of the sections like Android, Multiplatform, MySQL, etc., I guess also broken links)

Reply:
Is that because I did not generate javadocs before deploying this branch on my fork?

Reply:
yea, you have to run `/.github/workflows/prepare_mkdocs.sh` before running mkdocs. i've updated this branch with all the feedback now